lib/godwoken_explorer/udt.ex

```diff
@@ -75,9 +75,20 @@ defmodule GodwokenExplorer.UDT do
     end
   end
 
-  def count_holder(udt_id) do
-    from(cbub in CurrentBridgedUDTBalance, where: cbub.udt_id == ^udt_id)
-    |> Repo.aggregate(:count)
+  def count_holder(udt) do
```

Comment:
If there are a bridged token of id `1` and its mapped native token of id `2`, will they return different values?

Reply:
I think is better to return same value for bridge udt.

Reply:
so, does  bridged token should return it's mapping native token `holders count` ? @Keith-CY @zmcNotafraid